yt_dlp/extractor/fc2.py

```diff
@@ -22,8 +22,24 @@ class FC2IE(InfoExtractor):
         'md5': 'a6ebe8ebe0396518689d963774a54eb7',
         'info_dict': {
             'id': '20121103kUan1KHs',
-            'ext': 'flv',
             'title': 'Boxing again with Puff',
+            'ext': 'mp4',
+            'thumbnail': r're:https?://.+\.jpe?g',
+        },
+        'file_minsize': 633,
```

Comment:
I keep suggesting to remove this and you're not removing it. Is there a reason why we need this? We are skipping the download so it's pointless AFAIK
```suggestion
```